service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -106,7 +109,10 @@ public void configPush(CommandRequest request) {
                 for (Command command : bundle.getCommands()) {
                     Path executionLog = executeCommand(request, command, envVars);
                     if (executionLog != null) {
-                        streamCommandExecutionLogToEpCore(request, command, executionLog);
+                        if (commandLogStreamingProcessorOpt.isPresent()) {
+                            streamCommandExecutionLogToEpCore(request, command, executionLog);
+                        }
```

Comment:
Only stream if the commandLogStreamingProcessorOpt is present (e.g. in self managed mode)